spec/presenters/content_change_presenter_spec.rb

```diff
@@ -44,6 +55,27 @@
       expect(result).to eq(expected.strip)
     end
 
+    context "when subscriber list is for a content id" do
+      let(:subscriber_list) { create(:subscriber_list, content_id: SecureRandom.uuid) }
+
+      it "generates a url with utm_list_type=single-page" do
```

Comment:
This test name is wrong, should be `utm_campaign=govuk-notifications-single-page`

Reply:
ah, great spot - missed that in the title.
Done!